OtripleS.Web.Api/Services/Foundations/Teachers/TeacherService.Validations.cs

```diff
@@ -93,8 +93,8 @@ private static void Validate(params (dynamic Rule, string Parameter)[] validatio
 
         private static dynamic IsInvalidX(TeacherStatus status) => new
         {
-            Condition = status != TeacherStatus.Active,
-            Message = "Value is invalid"
+            Condition = Enum.IsDefined(status) is false,
```

Comment:
@hassanhabib I think we need two tests:
1. Teacher is added as Inactive so we say 'Value is invalid'
2. Techer is added with unrecognized status like the test here.

Reply:
@ShriHumrudha We can't put the condition of `Value is invalid` yet until we identify first whether this is a migration effort or an actual business flow. We should only prevent unrecognized values for now.